src/django/api/views/facility/facility_list_view_set.py

```diff
@@ -215,6 +220,8 @@ def create(self, request):
 
         csvreader = csv.reader(header.split('\n'), delimiter=',')
         for row in csvreader:
+            # TODO: remove create_nonstandard_fields after removing
```

Comment:
I think this comment can be deleted since the `def create` method will be removed soon. The `def createlist` method will be renamed to `def create` and will become the default POST handler for the `FacilityListViewSet`.

Reply:
I added a clearer explanation to this TODO comment, emphasizing that the `create_nonstandard_fields` function `(api/views/fields/create_nonstandard_fields.py)` needs to be removed after eliminating the `create` endpoint. This function isn't used anywhere and has been relocated to the `ProcessingFacility` class as a method.

Reply:
Ok, let's leave it then.

Reply:
I suppose this comment will help not to forget to remove this outstanding function.